lib/actions/api.js

```diff
@@ -214,6 +220,26 @@ function constructRoutingQuery (otpState, ignoreRealtimeUpdates) {
   // Check for routingType-specific API config; if none, use default API
   const rt = config.routingTypes && config.routingTypes.find(rt => rt.key === routingType)
   const api = (rt && rt.api) || config.api
+
+  // FIXME: Remove this block below once there is a fix
+  // for https://github.com/ibi-group/otp-middleware/issues/26.
```

Comment:
I'm going to add the blocked label until that issue is resolved then. We shouldn't create PRs with FIXMEs and instead should fix the underlying issues.

Reply:
I have reverted the snapshot. It's a good ideas to avoid changes if the file is irrelevant.

Reply:
As discussed, ibigroup/otp-middleware#26 will be worked on. Discuss if we need to show the working stop/trip viewer along with logging trip requests meanwhile.